### ISSUE TITLE
Broken link to /docs/ontology/

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/geneontology.github.io#96]
TLDR; link on ontology-documentation.md to ontology.md isn't working even though I copy/pasted the permalink. 

On https://github.com/geneontology/geneontology.github.io/blob/master/_docs/ontology-documentation.md (https://github.com/geneontology/geneontology.github.io/blob/master/_docs/ontology-documentation.md) 
under "GO terms", there's a link to 'here' (https://github.com/geneontology/geneontology.github.io/blob/master/docs/ontology).

https://github.com/geneontology/geneontology.github.io/blob/master/_docs/ontology.md (_docs not docs as above) lists the permalink as /docs/ontology/, what am I doing wrong?

**Comment by lpalbou** [comment]
Hi Suzy, nothing is broken and your link does work, however you can only test the links through the website itself and not through the GitHub preview of your markdown page.

In this case, go to your ontology-documentation page on the website:
[https://geneontology.github.io/docs/ontology-documentation/](https://geneontology.github.io/docs/ontology-documentation/)

If you test your "here" link, you'll see it does redirect to the correct page
